blueoil/datasets/ilsvrc_2012.py

```diff
@@ -72,17 +71,14 @@ def _init_files_and_annotations(self):
     def _files_and_annotations(self):
         txt_file = self.texts[self.subset]
 
-        df = pd.read_csv(
-            txt_file,
-            delim_whitespace=True,
-            header=None,
-            names=['filename', 'class_id'])
+        files, labels = list(), list()
+        with open(txt_file) as f:
+            for line in f:
+                items = line.split()
+                files.append(items[0])
+                labels.append(int(items[1]))
```

Comment:
Why we should cast to `int` ?

Reply:
The labels is taken as int in the implementation of pandas dataframe.
In my codes, the labels data is read in as string initially, I need to cast it to int to match with the consequent data handling.